engine/src/main/java/org/terasology/logic/behavior/GroupMindComponent.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.logic.behavior;
+
+import org.terasology.entitySystem.Component;
+import org.terasology.entitySystem.entity.EntityRef;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class GroupMindComponent implements Component {
```

Comment:
Is it possible to document this component now, or will it be done later?

Reply:
Sorry, didn't realize it was missing - fixed with additional commit.